### PR TITLE
misc

### PR DESCRIPTION
**Pull request by dhruvsol**

### CODE REVIEW

File: apps/api/src/worker.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+const STREAM_NAME = "cubik_events";
+
+async function main() {
+  let lastId = "$";
+
+  const keypair = web3.Keypair.generate();
+  const wallet = new Wallet(keypair);
+  const connection = new web3.Connection(
+    "https://devnet.helius-rpc.com/?api-key=d17f1226-b554-443e-978e-b41d01825ea4",
```
Comment (dhruvsol) [inline apps/api/src/worker.ts]: Can you move this to env

File: apps/api/src/worker.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    "confirmed",
+  );
+  const PROGRAM_ID = new web3.PublicKey(
+    "H8cQEqKuCU7uK2211tdEJ8pPFiT8L114Dj1d36LqeE5i",
```
Comment (dhruvsol) [inline apps/api/src/worker.ts]: Move to ENV

**Comment by dhruvsol** [comment]
@sahilpabale  Can you do these 2 small changes

**Comment by sahilpabale** [comment]
shooot my bad wait

**Comment by dhruvsol** [comment]
any update on this ??

**Comment by sahilpabale** [comment]
> any update on this ??

done. pushed

**Pull request merged**